note/規格書/03_事件評估與記憶.md

```
@@ -91,14 +94,14 @@ updated: 2026-08-12
 
 **由 LLM 自行評分。** 引擎不做加權——不同角色對同一事件的在意度差異，交由 AI 自行判斷。
 
-### 評分指示（寫入睡眠反思的呼叫中）　【推測待確認】
+### 評分指示（寫入睡眠反思的呼叫中）
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)